test/test_web.py

```diff
@@ -159,13 +161,15 @@ async def test_project_summary(self):
         expected = ProjectSummary(
             total_samples=1,
             total_participants=1,
-            sequence_stats={
+            total_sequences=1,
+            cram_seqr_stats={
                 'genome': {
                     'Sequences': '1',
                     'Crams': '0',
                     'Seqr': '0',
                 }
             },
+            sequence_stats={'genome': {'null': '1'}},
```

Comment:
Would you consider modifying this test to check if the batch stats come through correctly? I should have done the same with the `seqr`, because it obviously doesn't work for seqr.

Reply:
How detailed do you want me to go?

Reply:
Not super detailed, just set batch on one should basically test all of it (except the future case of no known batches results empty here.

Reply:
Fixed